Assets/SEE/Utils/VersionControlSystems.cs

```diff
@@ -0,0 +1,144 @@
+using UnityEngine;
+using LibGit2Sharp;
+using System.Linq;
+using System.IO;
+
+/// <summary>
+/// Contains all version control systems as subclasses, which derives from IVersionControl,
```

Comment:
What do you mean by _Contains_? It looks to me, this class should rather be a namepace.

Reply:
It is fixed.